content/smartmodules/certified/json-sql.md

```diff
@@ -0,0 +1,164 @@
+---
+title: JSON to SQL Mapping
+menu: Json-Sql
+weight: 20
+---
+
+This is certified by InfinyOn [map-type]({{<ref "../transform/map.md" >}}) SmartModule that converts arbitrary JSON records into [SQL model](https://github.com/infinyon/fluvio-connectors/blob/main/rust-connectors/models/fluvio-model-sql), which is a self-describing representation of SQL INSERT statements. Usually used in pipelines with [SQL Sink Connector][sql-sink-connector], the latter executes statements from the model against configured SQL database.
```

Comment:
~Usually used in pipelines with [SQL Sink Connector][sql-sink-connector], the latter executes statements from the model against configured SQL database.~ -> This SmartModule is intended to be used in [SQL Sink Connector][sql-sink-connector], to execute a command in a SQL database.